diff -ru mg3a-build/basic.c /Users/paag/Devel/mg3a/basic.c
--- mg3a-build/basic.c	2020-11-22 12:58:35.000000000 +0100
+++ /Users/paag/Devel/mg3a/basic.c	2020-11-27 17:27:38.000000000 +0100
@@ -630,7 +630,8 @@
 			 */
 			curwp = wp;
 			curbp = wp->w_bufp;
-			return showbuffer(curbp, curwp);
+			/* return showbuffer(curbp, curwp); */
+			return TRUE;
 		}
 	}
 	return FALSE;
diff -ru mg3a-build/def.h /Users/paag/Devel/mg3a/def.h
--- mg3a-build/def.h	2020-11-22 12:58:35.000000000 +0100
+++ /Users/paag/Devel/mg3a/def.h	2020-11-27 18:04:23.000000000 +0100
@@ -677,7 +677,11 @@
 void	invalidatecache(void);
 void	reframe_window(WINDOW *wp);
 void	update(void);
-void mouse_mode(int);
+
+#ifndef  MOUSE
+inline  static void mouse_mode(INT mode) {
+}
+#endif

 /* echo.c */

diff -ru mg3a-build/keymap.c /Users/paag/Devel/mg3a/keymap.c
--- mg3a-build/keymap.c	2020-11-22 12:58:35.000000000 +0100
+++ /Users/paag/Devel/mg3a/keymap.c	2020-11-27 17:27:38.000000000 +0100
@@ -467,7 +467,11 @@
 	1,
 	rescan,
 	{
+#ifdef MOUSE
 		{'A',	'M',	metalbA,	NULL},
+#else
+		{'A',	'D',	metalbA,	NULL},
+#endif
 	}
 };

diff -ru mg3a-build/tty.c /Users/paag/Devel/mg3a/tty.c
--- mg3a-build/tty.c	2020-11-22 12:58:35.000000000 +0100
+++ /Users/paag/Devel/mg3a/tty.c	2020-11-27 18:06:52.000000000 +0100
@@ -17,7 +17,10 @@
 int		setupterm(char *, int, int *);
 int		tputs(const char *, int, int (*)(int));

-static void ttputsf(char *format, ...);
+#ifdef MOUSE
+static void mouse_mode(INT);
+static void ttputsf(char *, ...);
+#endif

 char	*CM,			/* cursor move				*/
 	*CE,			/* clear to end of line			*/
@@ -328,12 +331,11 @@
  * mode  0 => disable mouse reporting
  *       1 => enable mouse reporting
  */
-void
-mouse_mode(int mode)
-{
 #ifdef MOUSE
-  ttputsf("\e[?9%c", (mode == 1) ? 'h' : 'l');  // Enable/Disable mouse report
-#endif
+static void
+mouse_mode(INT mode)
+{
+  ttputsf("\e[?1000%c", (mode == 1) ? 'h' : 'l');  // Enable/Disable mouse report
 }

 static void
@@ -347,3 +349,4 @@
   for (int i=0; localbuf[i]; i++)
     ittputc(localbuf[i]);
 }
+#endif
